sdemarre/diophantine_system/diophantine_system.asd

```diff
@@ -0,0 +1,13 @@
+(defsystem diophantine-system
+	   :defsystem-depends-on ("maxima-file" "info-index")
+	   :name "diophantine-system"
+	   :maintainer "Serge De Marre"
+	   :author "Serge De Marre"
+	   :licence ""
+	   :description "diophantine system of linear equations solver"
+	   :long-description "Maxima program to solve a system of linear diophantine equations A.x=b with integer elements of A,x and b."
+
+	   :components
+	   ((:info-index "diophantine-system-index")
```

Comment:
this is a reference to a non existing file, is there some tool to generate this file from the texi file ?